### ISSUE TITLE
BTC/USD seems to not be up to date

### ISSUE THREAD

**Issue by ismyhc** [bitpay/copay#6559]
Forgive me if this is a foolish observation since I'm new to bitcoin, but the usd value of the amount of BTC I have shows about $20 lower in bitpay than if I put the same amount of BTC into a realtime calculator.  Is this normal?

**Comment by ismyhc** [comment]
To be clear I do see the usd value in bitpay fluctuatating but it's always about 20$ less than I see in other BTC/USD calculators

**Comment by dabura667** [comment]
Welcome to Bitcoin.

Especially in times of great volatility (today was one of those days :-D ) The exchanges across the world, and even within the SAME COUNTRY can vary greatly in price moment to moment.

Currently Gemini says $3734.70 but bitpay's price index says $3715.16 which is a $20 difference but percentage wise it's only 0.5% (half a percent)

This is why when you are doing deals with people, it's a good idea to set a rate explicitly. "Let's trade based on Gemini's rate at the time of sale" etc.

As to why Bitpay is generally lower than what you see, Bitpay is not an exchange, Bitpay get's their price by averaging the prices of certain exchanges and weighting the average based on trade volume.

Online calculators will probably just look at one exchange. Which can be higher during a boom, and lower during a bust. (The lower volume is the more extreme price spikes are, which are eventually equalized by people with accounts on multiple exchanges trading the difference across other high volume exchanges)

tl;dr All currencies do this. If you google "1 USD in Japanese Yen" the rate you get is different than the airport, and is different than if you created a brokerage for foreign currency exchanging.

**Comment by dabura667** [comment]
This is also good to keep in mind when buying things online.

If Bitpay is the payment processor for the store, you will buy using Bitpay's rate.

If Coinbase you will use Coinbase's rate. etc.

So your wallet app's rate won't necessarily be exactly the same as the payment processor's rate.

**Comment by ismyhc** [comment]
Okay got it! Makes sense now :). I really appreciate the detailed explanation. I transferred from Coinbase wallet to my bitpay because everything I read said it was safer and I kept hearing bad things about keeping your bitcoin on Coinbase. I do miss Coinbases built in graphs though. Something bitpay could add to make it more attractive, especially to new crypto currency users!

Thanks again fot your help;)
